mail/internal/mail/smtp.go

```diff
@@ -199,7 +199,11 @@ func (d *Dialer) DialAndSend(m ...*Message) error {
 	}
 	defer s.Close()
 
-	return Send(s, m...)
+	if errs := Send(s, m...); len(errs) > 0 {
```

Comment:
as long as there are messages, the length of errs will always be > 0 (example: https://play.golang.org/p/yEQg769VNmz )

Reply:
indeed, thank you for the response. Resolved.